src/Spanner/Session/CacheSessionPool.php

```diff
@@ -777,4 +764,63 @@ private function validateConfig()
             );
         }
     }
+
+    /**
+     * Delete the provided sessions.
+     *
+     * @param array $sessions
+     */
+    private function deleteSessions(array $sessions)
+    {
+        $calls = [];
+
+        foreach ($sessions as $session) {
+            $calls[] = $this->database->connection()
+                ->deleteSessionAsync([
+                    'name' => $session['name'],
+                    'database' => $this->database->name()
+                ]);
+        }
+
+        foreach ($calls as $call) {
+            $call->wait();
```

Comment:
What exactly does it mean that the delete RPC will not finish? Will gRPC explicitly cancel the RPC?